tox.ini

```diff
@@ -11,11 +11,15 @@ deps =
 commands = py.test
 
 [pytest]
-addopts=--cov=uswds_forms --cov-report=term --cov-report=html
+addopts=--cov=uswds_forms --cov-report=term --cov-report=html --ignore=venv
 
 [testenv:bandit]
 deps = bandit
-commands = bandit -r {toxinidir}/uswds_forms -x uswds_forms/tests
+
+# The inclusion of multiple identical exclusion directories with path
+# separators for Windows + UNIX is super weird but seems like the only
+# way to make this command work on both platforms.
+commands = bandit -r {toxinidir}/uswds_forms -x uswds_forms/tests,uswds_forms\tests
```

Comment:
Oh yeah this does look weird. I don't have access to a Windows right now to test, so I don't know if there's a better way. I'm fine with this as is, though I might revisit it later when I've got a Windows and try to see if I can find a better way. (Or, you know, we could just strip Bandit; I'm not convinced it provides a ton of value. I'm mostly including it because why not.)